src/components/endowed-positions/EndowedPositionsFooter.js

```diff
@@ -6,7 +6,7 @@ const EndowedPositionsFooter = () => (
       <p>
         This site is a comprehensive directory of endowed positions at Stanford.
         It is updated six times per year, following meetings of Stanford&apos;s
-        Board of Trustees. Last updated April 29, 2024. For questions or
+        Board of Trustees. Last updated August 1, 2024. For questions or
```

Comment:
Haven't we already merged this change into `dev`, @wmui51 ?

Reply:
@yvonnetangsu i think so, i am not sure why these changes are still showing, let me try rebasing from dev.

Reply:
i have rebased, and it looks like the previous commit is no longer there.